### PR TITLE
created mentor-branch

### PR DESCRIPTION
**Pull request by VolodymyrBondarchuk**
Please review

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Espress0JavaTeam/AirLine/pull/1" target="_blank">ReviewNinja

File: readme.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #Here should be description of the project
```
Comment (VolodymyrBondarchuk) [inline readme.md]: Please write description here

File: readme.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
 
-Testing review ninja
\ No newline at end of file
```
Comment (VolodymyrBondarchuk) [inline readme.md]: Remove this line

File: readme.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
 #Here should be description of the project
 
-Testing review ninja
```
Comment (VolodymyrBondarchuk) [inline readme.md]: good

**Pull request merged**